script.js

```diff
@@ -1,4 +1,190 @@
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first.
+// You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+// Score
+// Keep score for each player. The score is the running sum of all numbers that player has generated so far. This means there is no permanent winner, only a temporary leader.
+
+// global variables
+var playerOneDiceOne = "";
+var playerOneDiceTwo = "";
+var playerTwoDiceOne = "";
+var playerTwoDiceTwo = "";
+var playerOneNumber = "";
+var playerTwoNumber = "";
+var playerOneScore = [];
+var playerTwoScore = [];
+var gameMode = "player 1";
+
+// calc total player 1 score indefinitely
+var totalPlayerScore = function () {
+  var playerOneTotalScore = 0;
+  var playerTwoTotalScore = 0;
+
+  for (i = 0; i < playerOneScore.length; i++) {
+    playerOneTotalScore = playerOneScore[i];
+
+    console.log(
+      playerOneScore
+        .map(function (elt) {
+          // assure the value can be converted into an integer
+          return /^\d+$/.test(elt) ? parseInt(elt) : 0;
+        })
+        .reduce(function (a, b) {
+          // sum all resulting numbers
+          return a + b;
+        })
+    );
```

Comment:
i think we have resolved this part in class so it works now. The additional comment is to remove console.logs from submitted code as it makes the code longer and that it is not part of the functionality of the code